tests/data/state_estimation/current-sensor/basic-local-current-sensor/sym_output.json

```diff
@@ -0,0 +1,75 @@
+{
+  "version": "1.0",
+  "type": "sym_output",
+  "is_batch": false,
+  "attributes": {},
+  "data": {
+    "node": [
+      {
+        "id": 1,
+        "energized": 1,
+        "u_pu": 0.9999918620376325,
+        "u": 9999.918620376324,
+        "u_angle": 1.222133428418590e-13
```

Comment:
why not just make it `0` by means of `rtol` and `atol`? or is this the exact solution?